ccdproc/image_collection.py

```diff
@@ -710,6 +710,8 @@ def _find_keywords_by_values(self, **kwd):
                         pattern = re.compile(value,
                                              flags=re.IGNORECASE)
                     else:
+                        # Add escape characters to parentheses:
+                        value = value.replace('(',r'\(').replace(')',r'\)')
```

Comment:
I think this would fix #769 it were this instead, so that all special characters were escaped:

```suggestion
                        value = re.escape(value)
```